runner/docker-shim.sh

```diff
@@ -2,11 +2,16 @@
 
 set -Eeuo pipefail
 
+DOCKER=/usr/bin/docker
+if [ ! -e $DOCKER ]; then
+  DOCKER=$HOME/bin/docker
+fi
```

Comment:
could this be simplified with `DOCKER=$(which docker)`?

Reply:
Maybe not, because it would resolve to `/usr/local/bin/docker` (otherwise this shim won't work) which results in an infinite loop.